Makefile

```diff
@@ -161,7 +151,8 @@ check-tidy:
 
 # Serve docs
 .PHONY: docs
-docs: godoc
+docs: $(TOOLDIR)/godoc
+	$(info serviing docs on http://127.0.0.1:6060/pkg/$(GOMODNAME)/)
```

Comment:
```suggestion
	$(info serving docs on http://127.0.0.1:6060/pkg/$(GOMODNAME)/)
```

Reply:
LOL... I might have this typo in a few projects 🤦‍♂️